### ISSUE TITLE
typo

### ISSUE THREAD

**Issue by aripakman** [jcjohnson/pytorch-examples#22]
when it says:
"after backpropagation x.grad will be another Tensor holding the gradient of x with respect to some scalar value."

it should be 
"after backpropagation x.grad will be another Tensor holding the gradient of some scalar value with respect to x."

**Comment by jcjohnson** [comment]
I think this is now fixed.
